### PR TITLE
testing new docs folder

### PR DESCRIPTION
**Pull request by adlondon**
✔︎ I've tested in all browsers (IE11, Firefox, Safari, Chrome, and iOS)

### What should this PR do?
Created docs directory

### CODE REVIEW

File: src/pages/animations.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 
 <section class="site-section">
-  <h3 class="site-subheadline">Animation Types</h3>
+  <h3 class="site-subheadline">Animation Tips</h3>
```
Comment (alexgsummer) [inline src/pages/animations.html]: Did you mean to change this to Tips?

File: package.json [hunk 2]
```diff
@@ -33,10 +33,9 @@
     "cssnano": "3.10.0",
     "del": "3.0.0",
     "eslint-config-rhinogram": "0.0.10",
-    "gulp": "3.9.1",
+    "gulp": "^3.9.1",
```
Comment (alexgsummer) [inline package.json]: We prefer to lock down our dependencies to specific versions, so here remove the carets (^).

Comment (adlondon) [reply package.json]: Haha, was testing to make sure changes were coming through. Ill fix that right quick

File: src/pages/animations.html (hunk 1 above)
Comment (adlondon) [reply src/pages/animations.html]: Haha, was testing to make sure changes were coming through. Ill fix that right quick

**Pull request merged**